### PR TITLE
Added Codeception project

### PR DESCRIPTION
**Pull request by javiereguiluz**
The description in the `projects/en/codeception.html.twig` file should be probably improved. Sadly, I didn't find a simple paragraph describing the project in the Codeception website.

### CODE REVIEW

**Comment by DavertMik** [comment]
Good point, looks like I need to update description on site. 

Please update to:

Codeception is a full-stack testing framework which incorporates acceptance, functional, and unit testing. It uses simple PHP DSL to describe test scenarios and includes over 20 modules to solve common problems. With Codeception you can get test executed inside every popular PHP framework, use PhpBrowser (based on Guzzle) or run acceptance test with Selenium WebDriver.

**Comment by javiereguiluz** [comment]
@DavertMik thanks for the description. It's much better now. I consider this pull request finished, so I'll wait for community review and then, I'll merge it quickly.

File: projects/en/codeception.html.twig [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{% block description %}
+    <p>
+
+        Codeception is a full-stack testing framework which incorporates
+        acceptance, functional, and unit testing. It uses simple PHP DSL to
```
Comment (xabbuh) [inline projects/en/codeception.html.twig]: [...] uses a simple [...]

File: projects/en/codeception.html.twig [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+        describe test scenarios and includes over 20 modules to solve common
+        problems. With Codeception you can get test executed inside every
```
Comment (xabbuh) [inline projects/en/codeception.html.twig]: tests

File: projects/en/codeception.html.twig [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+        popular PHP framework, use PhpBrowser (based on Guzzle) or run
```
Comment (xabbuh) [inline projects/en/codeception.html.twig]: missing comma before "or"

File: projects/en/codeception.html.twig [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+        acceptance test with Selenium WebDriver.
```
Comment (xabbuh) [inline projects/en/codeception.html.twig]: tests

**Comment by xabbuh** [comment]
@DavertMik Can you please have a look if you agree with the remarks I made?

**Comment by DavertMik** [comment]
Text looks good. 
Could you include this pic as a logo? http://codeception.com/images/logo.png

**Comment by javiereguiluz** [comment]
I've just updated the logo as requested. Everything is ready so I'm going to merge it. Thanks!

**Pull request closed without merging**